### ISSUE TITLE
Upload : server respond with status 417 Expectation Failed

### ISSUE THREAD

**Issue by MaherFa** [orgrimarr/node-wetransfert#21]
Hello! I'm trying to initiate a simple upload and getting the following error:

```
Info: Start process (10:32:52 PM)
Error: 
ERROR Error: Error GET https://wetransfer.com/api/v4/auth/session server respond with status 417 Expectation Failed. ["message", "CSRF token was invalid"]
    at Object.exports.login (c:\maher_scripts\weTransfer\node_modules\wetransfert\utils\utils.js:192:15)
    at process._tickCallback (internal/process/next_tick.js:68:7)

Error: 
ERROR Error: Error GET https://wetransfer.com/api/v4/auth/session server respond with status 417 Expectation Failed. ["message", "CSRF token was invalid"]
    at Object.exports.login (c:\maher_scripts\weTransfer\node_modules\wetransfert\utils\utils.js:192:15)
    at process._tickCallback (internal/process/next_tick.js:68:7)

Info: End process (10:33:02 PM)
```
here's my code with my private data replaced:

```
const { upload } = require('wetransfert');
const myUpload = upload('myEmail@gmail.com', ['myEmail@gmail.com'], ['C:/example/example.txt'], 'Hello World', 'en', 'myEmail@gmail.com', 'myPassword')
.on('progress', (progress) => console.log('PROGRESS', progress))
.on('end', (end) => console.log('END', end))
.on('error', (error) => console.error('ERROR', error));

setTimeout(function(){
    myUpload.cancel();
}, 10000);
```

**Comment by orgrimarr** [comment]
Hi,
Thanks for your issue.
I have fix the issue and publish a 2.3.2 release. https://www.npmjs.com/package/wetransfert

Can you upgrade and try again ?

**Comment by MaherFa** [comment]
thank you very much for the quick fix.
2.3.2 resolved the issue
